Assets/Editor/UnitTests/Models/ScheduledEventTest.cs

```diff
@@ -17,6 +17,7 @@ public class ScheduledEventTest
 {
     private Action<ScheduledEvent> callback;
     private bool didRun = false;
+    private int runCount = 0;
```

Comment:
Good point... Though I'd also prefer it be in another PR... done by someone else... mostly because I'm not absolutely clear on the functioning of the scheduler at this time... I only figured out enough to get what I needed for autosaving to work the way I thought it should.